### DISCUSSION TITLE
Our first beta release is now online!

### DISCUSSION THREAD

**Discussion by DeeJayTC** [WeAreDotnet/dotnet-creator-cards#65] [Announcements]
Hey Everyone! 
Some great news. 

We have our first beta version online!
Go to https://www.wearedotnet.io and check it out!

Each Creator now has their own little profile page similar to this one:
![image](https://user-images.githubusercontent.com/4077759/224432029-98a77721-cbc6-4afc-acd8-7590570c9c8c.png)

Just click on any creator card you see to get there. 
We are already parsing the creator feeds and always list the last 30 entries we have on the landing page. 
![image](https://user-images.githubusercontent.com/4077759/224432311-688c494b-0d5e-43e8-ab7c-d0e7ed0b74b0.png)

There's way more to come!

Stay tuned!

As a side note, this is a beta version, there are bugs for sure but don't hesitate to let us know on whatever channel.

**Comment by TechWatching** [comment]
I was just wondering hwo were you parsing the creator feeds. Are you doing it on a schedule? I asking the question because my feed was broken, I have just fixed it, but the content on my creator page is still empty.

**Comment by mikeirvingweb** [comment]
How do we submit issues? I've noticed the feed on a user page is in a messed up order

Mine lists

2022 items
2023 items
2021 items

Ideally it should be date descending, 2023, 2022, 2021, IMO 🙂

see: https://www.wearedotnet.io/mikeirvingweb

**Comment by ZuraizAhmedShehzad** [comment]
are you guys planning to make this platform opensource at some point?
